Ontology/Willow/Capability/Actuator/Binary Actuator/OnOff Actuator/Cooling/Stages/CoolingStage9Actuator.json

```diff
@@ -0,0 +1,16 @@
+{
+  "@id": "dtmi:com:willowinc:CoolingStage9Actuator;1",
```

Comment:
This is getting crazy in terms of needing a separate model for every stage. Should we consider using a property on the twin instead? We do already have this concept as if we want to consider that again: categorizationProperty.stage